content/javascript/concepts/dates/terms/getFullYear/getFullYear.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: 'getFullYear()' 
+Description: 'Return the current full year' 
+Subjects: 
+  - 'Web Development'
+Tags: 
+  - 'Date'
+  - 'Method'
+CatalogContent: 
+  - 'paths/web-development'
+---
+
+The **getFullYear** method returns the current four digit year
+
+## Syntax
+
+```js
+myDate.getFullYear()
+```
+
+## Example
+
+```js
+const getYear = new Date().getFullYear()
```

Comment:
```suggestion
const getYear = new Date().getFullYear()
```
Change the variable name so it isn't the name of another JavaScript date method.

Reply:
Good point I have changed this